spec/dataframe_spec.rb

```diff
@@ -1820,6 +1820,33 @@
     end
   end
 
+  describe 'uniq' do
+    let(:df) do
+      Daru::DataFrame.from_csv 'spec/fixtures/duplicates.csv'
+    end
+
+    context 'with no args' do
+      it do
+        result = df.uniq
+        expect(result.shape.first).to eq 30
+      end
+    end
+
+    context 'given a vector' do
+      it do
+        result = df.uniq("color")
+        expect(result.shape.first).to eq 2
+      end
+    end
+
+    context 'given an array of vectors' do
+      it do
+        result = df.uniq(["color", "director_name"])
```

Comment:
Why not `uniq(:color, :director_name)` (e.g. splatting arguments)? It looks more reasonable for me.

Reply:
Mostly just because the `group_by` takes an array but both work for me. Have changed.